app/components/TopBar/TopBar.tsx

```diff
@@ -1,9 +1,15 @@
 import React from 'react';
 import { Toolbar, Container, makeStyles } from '@material-ui/core';
+import { useRecoilValue } from 'recoil';
+
+import { alphaState } from '../../store';
 
 import StartGameButton from './StartGameButton';
+import StartAlphaButton from './StartAlphaButton';
 import AppTabs from './AppTabs';
+import { tabList } from './AppTabs';
 import LoadingSuspense from '../LoadingSuspense';
+import { globalText } from '../../helpers/static-text';
```

Comment:
you added some imports that you don't use. Would have been caught by either ESLint or just normal TypeScript warnings.